happi/backends/json_db.py

```diff
@@ -75,8 +75,8 @@ def _load_or_initialize(self):
         return self._load_cache
 
     @property
-    def all_devices(self):
-        """All of the devices in the database."""
+    def all_items(self):
```

Comment:
From first look, it seems like the approach here is to rename the backend methods and deprecate the client methods- is that correct? I think it's a good approach.

Reply:
Yes, that was the intent, at least. Hopefully it's consistent here.

Reply:
Reading through very carefully, I think it is